src/components/EmployeesList/EmployeesList.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import Paper from '@mui/material/Paper';
+import { v4 as uuidv4 } from 'uuid';
+
+export default function EmployeesList() {
+  function createData(
+    name,
+    department,
+    position,
+    email,
+    mobPhone,
```

Comment:
```suggestion
    mobilePhone,
```
надо писать без сокращений, кроме может быть самых распространенных http, html, xml. Иначе тяжело запомнить где есть сокращение, а где его нет (например internalPhone уже без сокращения написан).